ollama/_types.py

```diff
@@ -229,12 +230,14 @@ class Function(SubscriptableBaseModel):
     description: str
 
     class Parameters(SubscriptableBaseModel):
-      type: str
+      type: Union[str, List[str]]
       required: Optional[Sequence[str]] = None
       properties: Optional[JsonSchemaValue] = None
 
     parameters: Parameters
 
+    return_type: Optional[Union[str, List[str]]] = None
```

Comment:
It is not but is built into other libs - thought we should have it exposed in case we need it later but happy to take out. Don't have a strong opinion here

Reply:
Decided to rm this. We can easily bake in anytime if needed